src/utils/problemUtils.ts

```diff
@@ -0,0 +1,8 @@
+import kebabCase = require("lodash.kebabcase");
+import { IProblem, langExt } from "../shared";
+
+export function genFileName(node: IProblem, language: string): string {
+    const name: string = kebabCase(node.name);
+    const ext: string | undefined = langExt.get(language);
+    return `${node.id}.${name}.${ext}`;
```

Comment:
`ext` here might be `undefined`. Though according to current logic, it's impossible.

I think you can have another util method here to parse the `ext` from a language name. Meanwhile, throw an error if the language is not supported.

Reply:
OK. I'll fix it.